src/components/app.tsx

```diff
@@ -1,35 +1,27 @@
 import { inject, observer } from "mobx-react";
 import * as React from "react";
-import { IAllStores } from "../index";
+import { IStores } from "../models/stores";
 import { authenticate } from "../lib/auth";
-import { UserModelType } from "../models/user";
 import { AppContainerComponent } from "./app-container";
 
 import "./app.sass";
 
-interface IInjectedProps {
-  user: UserModelType;
-  devMode: boolean;
+interface IProps {
```

Comment:
What do you think about exporting this so other components can just import it and not have to define it themselves?  If it was renamed `IStoreProps` other components that needed additional props could just use `interface IProps extends IStoreProps {...` but those that didn't could just use `IStoreProps` directly.

Reply:
I think exporting IStoreProps makes sense.